theme/src/ts/terraform-compare.ts

```diff
@@ -0,0 +1,25 @@
+import { gb } from "../../stencil/src/util/util";
+
+function initTerraformCompare() {
+    console.log("Terraform compare module initialized");
```

Comment:
We should probably remove this console log on the final version.

I've checked the version built on http://www-testing-pulumi-docs-origin-pr-13171-7992718f.s3-website.us-west-2.amazonaws.com/docs/iac/concepts/vs/terraform/ and i don't see this event in the console 🤔

Reply:
fixed!